cluster/gce/gci/configure-helper.sh

```diff
@@ -675,14 +725,22 @@ contexts:
     user: kube-apiserver
   name: webhook
 EOF
-    # This is the config for the image review admission controller.
-    cat <<EOF >/etc/admission_controller.config
+    GCP_IMAGE_REVIEW_CONFIG="imagePolicy:
 imagePolicy:
   kubeConfigFile: /etc/gcp_image_review.config
   allowTTL: 30
   denyTTL: 30
   retryBackoff: 500
-  defaultAllow: true
+  defaultAllow: true"
+    # Make sure this is included in the admission control config file
+    USING_ADMISSION_CONTROL_CONFIG_FILE=1
+  fi
+
+  if [[ -n "${USING_ADMISSION_CONTROL_CONFIG_FILE:-}" ]]; then
+    # This is the config for the admission controllers.
+    cat <<EOF >/etc/admission_controller.config
+${ADMISSION_CONTROL_CONFIG_PLUGINS:-}
```

Comment:
If we only use the config file for imagePolicy but not for the webhooks, will the ${ADMISSION_CONTROL_CONFIG_PLUGINS} miss the header:

```
kind: AdmissionConfiguration
 apiVersion: apiserver.k8s.io/v1alpha1
 plugins:
```

Reply:
That is how the code worked before so I thought it shouldn't be included unless there are other plugins besides imagePolicy